modules/integration/pages/ledger-quick-start.adoc

```diff
@@ -158,50 +157,158 @@ You are all caught up to block height 109
 * All transactions are stored in the ledger starting with the genesis initial state.
 * Tokens are entirely fungible.
-* Account identifiers are 32 bytes and, roughly speaking, are the hash of a public key (together with some additional sub-account specifier).
+* Account identifiers are 32 bytes and are roughly the equivalent of the hash of a public key, optionally together with some additional sub-account specifier.
 
-The ICP token differs from Bitcoin however in a few small ways.
-For example:
+The ICP token differs from Bitcoin in the following ways:
 
 * Rather than using proof of work, staked participant nodes use a variant of threshold BLS signatures to agree on a valid state of the chain.
 * Any transaction can store an 8-byte memo -- this memo field is used by the Rosetta API to store the nonce that distinguishes between transactions. However, other uses for the field are possible.
 
+== Frequently asked questions
+The following questions are taken from the most commonly reported questions and blockers from the developer community regarding Rosetta integration with the {IC}.
+
+=== The Rosetta node
+
+==== How to I run an instance of the Rosetta node?
+
+An easy way to accomplish this is to use the [`dfinity/rosetta-api`](https://hub.docker.com/r/dfinity/rosetta-api/tags?page=1&ordering=last_updated) Docker image. Once the node initializes and syncs all blocks, you can perform queries and submit transactions by invoking the Rosetta API on the node. The node listens on the `8080` port.
+
+==== How do I know if the node has caught up with the test net?
+
+Search the `Starting Rosetta API server` startup log. There will be a log entry that says `You are all caught up to block XX`. This message confirms that you are caught up with all blocks.
+
+==== How to persist synced blocks data?
+
+Mount the `/data` directory elsewhere.
+
+==== Is the Rosetta node versioned?
+
+Not yet. Before launch, when we push to the `dfinity/rosetta-api:latest` image, it's usually a major update that we'll announce in our communication channels beforehand.
+
+We'll soon implement nightly builds of the image, and CI will ensure it works before pushing. Other than `latest`, those images will also be tagged with the build date, so for more reproducibility, it's possible to use the image of a specific date tag rather than `latest`. We'll announce when nightly builds become available.
+
+==== How do I connect to the main net instead of the test net?
+
+Start `dfinity/rosetta-api` with `--help`, you can see some additional CLI arguments that can be passed. Among those there are `--canister-id` and `--ic-url` which can be used to configure the ledger destination. At the moment, they default to the test net.
+
+**Note**: The main net is not live yet; it will be live some time before the publicly announced date, and we'll push the updated image to point to the main net to ensure you can perform testing on the main net beforehand.
+
+=== ICPT-specific Rosetta API details
```

Comment:
Please do not use ICPT. It should only be ICP tokens.

Reply:
This is straight from the FAQ and I would not have understood. Thank you!